### PR TITLE
SensorSpeed: use the same intervals on all platforms

### PR DESCRIPTION
**Pull request by janusw**
### Description of Change ###

* the intervals are matching those of Android, see https://developer.android.com/guide/topics/sensors/sensors_overview
* note that we do not use a zero-interval for SensorSpeed.Fastest (since this might turn off the sensor, in particular on UWP); instead we use 5ms, which is roughly what I have measured on Android 10 for SensorSpeed.Fastest

### Bugs Fixed ###

- Related to issue #1438

### API Changes ###

None.

### Behavioral Changes ###

None.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [X] Rebased on top of `main` at time of PR
- [X] Changes adhere to coding standard
- [X] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### CODE REVIEW

File: docs/en/Xamarin.Essentials/SensorSpeed.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
       </ReturnValue>
       <MemberValue>3</MemberValue>
       <Docs>
-        <summary>Get the sensor data as fast as possible.</summary>
+        <summary>Get the sensor data as fast as possible (the atual rate is platform-dependent).</summary>
```
Comment (jamesmontemagno) [inline docs/en/Xamarin.Essentials/SensorSpeed.xml:50]: "the actual"

Comment (janusw) [reply docs/en/Xamarin.Essentials/SensorSpeed.xml]: Thanks for catching this one, James! Fixed with b6ea7e6e3beefcf0079bee1512f4565d42efcb1e.

**Comment by janusw** [comment]
@jamesmontemagno @mattleibow Any chance this might be considered for 1.7.0?

**Comment by janusw** [comment]
Rebased to 1.7.0.

Also: Ping! (This has been waiting for 9 months now ...)

**Comment by jfversluis** [comment]
I'm on the fence about this one to still include it in this stage of the Essentials lifetime. Technically it's a breaking change, we need to update docs, etc. As much as I love this to be consistent, I don't think we should do this anymore for Xamarin.Essentials.

I think this would be something nice to include in .NET MAUI though. Any thoughts @jamesmontemagno?

**Comment by janusw** [comment]
> I'm on the fence about this one to still include it in this stage of the Essentials lifetime. Technically it's a breaking change, we need to update docs, etc. As much as I love this to be consistent, I don't think we should do this anymore for Xamarin.Essentials.
> 
> I think this would be something nice to include in .NET MAUI though.

Thanks for the comment, @jfversluis. To be honest, I had basically given up on this PR already :/

I can certainly forward-port it to the Maui repo, if there is a chance of getting it approved there.

**Comment by jfversluis** [comment]
I know @janusw so sorry! We're doing better with .NET MAUI!

Let's not do this for Essentials anymore. It's a breaking change and a bigger one. I see you brought it to the .NET MAUI repo, thank you for that! So let's make sure we add it there.

Thank you for all the patience here, much appreciated!

**Pull request closed without merging**